drivers/led/Kconfig

```diff
@@ -27,6 +27,7 @@ config LED_SHELL
 	  Enable LED shell for testing.
 
 # zephyr-keep-sorted-start
+source "drivers/led/Kconfig.arduino"
```

Comment:
Since this is mostly "product name", `Kconfig.modulino` might be better.
It's not blocking.

Reply:
Good observation, changed it, matches the file naming I chose as well, added that path to the Arduino area in the maintainers file as well while at it, I missed that too.